src/Aquarius.Client/Samples/Client/ServiceModel.cs

```diff
@@ -3537,7 +3636,7 @@ public class ChartDataPoint
         public string MdlValueUnitCustomId { get; set; }
         public double DepthValue { get; set; }
         public string DepthUnitCustomId { get; set; }
-        public DetectionConditionType DetectionCondition { get; set; }
```

Comment:
# Form two of the breaking API change - Response property changes

JSON deserialization errors when receiving `CategoricalResult` or `NumericResult` objects, even if your code is ignoring those properties.



### Affect on running code based on the previous SDK: - Runtime exceptions

If your integration was receiving any JSON responses with `CategoricalResult` or `NumericResult` objects, the SDK's JSON deserialization logic will throw an error (because it was expecting a JSON string value starting with a double-quote, but instead encounters a JSON object starting with a left curly brace.

This is particularly nasty, since your code might have been issuing a `GET /api/v2/observations` request, but never actually inspecting these properties. That doesn't matter to the JSON deserializer, which will be confused and throw a `SerializationException`.

![image](https://user-images.githubusercontent.com/54773/97055365-85625580-153b-11eb-8a1a-5f0bf449c426.png)

### Affect on software after upgrading to SDK v20.3.1: - Maybe a compilation error, maybe not

If your code never actually accessed these properties, then it will recompile fine. But if it did, you will need to update your code.